gdal/swig/python/gdal-utils/osgeo_utils/gdal_proximity.py

```diff
@@ -48,15 +50,15 @@ def Usage():
 
 
 def main(argv):
-    frmt = None
+    driver = None
```

Comment:
driver_name would be more appropriate (driver could be confused with an object of type gdal.Driver) . Same for similar change in other utilities